### PR TITLE
Save, get player volume in localStorage

### PR DESCRIPTION
**Pull request by slaponicus**

### CODE REVIEW

**Comment by slaponicus** [comment]
I also added a 404 page with a bit of styling in this PR

**Comment by mcfiredrill** [comment]
Nice!

File: app/components/datafruits-player.js [hunk 1]
```diff
@@ -110,7 +110,10 @@ export default Component.extend({
         $('.jp-loading').hide();
       },
       solution: 'html, flash',
-      cssSelectorAncestor: ""
+      cssSelectorAncestor: "",
+	  volumechange: function(e){
```
Comment (mcfiredrill) [inline app/components/datafruits-player.js]: I think your editor might be using tabs? I try to keep it consistent across the codebase. No biggie tho

File: app/routes/not-found.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+export default Ember.Route.extend({
```
Comment (mcfiredrill) [inline app/routes/not-found.js]: Yeah the new syntax is as stickler says ::
```
import Route from '@ember/routing/route'

export default Route.extend({
```

Comment (slaponicus) [reply app/routes/not-found.js]: ah yeah my bad, let me fix all that!

**Comment by mcfiredrill** [comment]
Merged in https://github.com/datafruits/datafruits/pull/190

**Pull request merged**